Makefile

```diff
@@ -19,7 +19,7 @@ OBJ := $(SRC:.c=.o)
 
 # define default flags, and override to append mandatory flags
 CFLAGS ?= -O3 -Wall -Wextra -Werror
-override CFLAGS += -std=gnu99 -fPIC -Ilib/src -Ilib/include
+override CFLAGS += -std=gnu11 -fPIC -Ilib/src -Ilib/include
```

Comment:
Anonymous unions are a C11 extension. We could fix this without switching to a newer C version by giving these a name, but C11 is almost ten years old, so it seems fine to me?